source/user-manual/reference/tools/agent_upgrade.rst

```diff
@@ -9,6 +9,8 @@ agent_upgrade
 
 The agent_upgrade program allows you to list outdated agents and upgrade them.
 
+.. note:: In case of having a multi-node Wazuh cluster agent_upgrade must be executed on the node where the agent is connected.
```

Comment:
Maybe a `,` can be used after `Wazuh cluster`

Reply:
Done!